go.mod

```diff
@@ -166,9 +167,11 @@ require (
 // replace (
 // 	<temporary replace>
 // )
+replace cosmossdk.io/api => ./api
 
 // Below are the long-lived replace of the Cosmos SDK
 replace (
+	cosmossdk.io/x/protocolpool => ./x/protocolpool
```

Comment:
This should be a v0.51 blocker to have extracted gov and distribution so that the SDK does not depend on a module.

Reply:
i think we should be able to avoid this? lets do it in a follow up

Reply:
I could remove it by using a pseudo version if its fine
cc: @tac0turtle @julienrbrt